backend/epub/epub-document.c

```diff
@@ -181,7 +181,17 @@ epub_document_check_hits(EvDocumentFind *document_find,
 {
 	gchar *filepath = g_filename_from_uri((gchar*)page->backend_page,NULL,NULL);
 	htmlDocPtr htmldoc =  xmlParseFile(filepath);
+    if(htmldoc==NULL) {
+        g_free (filepath);
+        return 0;
+    }
 	htmlNodePtr htmltag = xmlDocGetRootElement(htmldoc);
+    if (htmltag == NULL){
```

Comment:
code formatting: blank space is missing, between the closing parenthesis and the opening curly bracket